script/setup-win-for-dev.bat

```diff
@@ -56,6 +56,7 @@ REM Install Windows SDK
 choco install windows-sdk-10-version-2104-all
 
 REM Install nodejs python git and yarn needed dependencies
+choco install -y --force nodejs@v18.12.1
```

Comment:
nodejs-lts on the next line should be removed.

Reply:
D'oh, missed that it was also there 🤦‍♀️ Thanks!